File: 0944-delete-columns-to-make-sorted/0944-delete-columns-to-make-sorted.py
class Solution:
    def minDeletionSize(self, strs: List[str]) -> int:
        cols = len(strs[0])
        ans = 0
        for i in range(cols):
            order = ord('a')
            for j in range(len(strs)):
                if order > ord(strs[j][i]):
                    ans += 1
                    break
                else:
                    order = ord(strs[j][i])
        return ans

File: 0944-delete-columns-to-make-sorted/0944-delete-columns-to-make-sorted.cpp
class Solution {
public:
    int minDeletionSize(vector<string>& strs) {
        // String length.
        int K = strs[0].size();
        
        // Variable to store the count of columns to be deleted.
        int answer = 0;

        for (int col = 0; col < K; col++) {
            for (int row = 1; row < strs.size(); row++) {
                if (strs[row][col] < strs[row - 1][col]) {
                    answer++;
                    break;
                }
            }
        }
        
        return answer;
    }
};